docs/CONTRIBUTING.md

```diff
@@ -669,126 +669,82 @@ Hello, World!
 You can start [IDE](https://github.com/enso-org/enso/tree/develop/gui) with a
 development version of the language server. IDE executable has
 `--external-backend` flag that switches off the bundled backend. That requires
-you to run the project manager process yourself. You can either get a project
-manager from one of the latest releases on
-[GitHub](https://github.com/enso-org/enso/releases), or build one using SBT
-`buildProjectManagerDistribution` command.
-
-Running development version of the IDE is possible via the `./run` script in the
-root of the repository:
+you to run the project manager process yourself. Running development version of
+the IDE is also possible via the `./run` script in the root of the repository:
 
 ```bash
-$ ./run ide start --wasm-profile dev --external-backend
+enso$ ./run gui watch --skip-wasm-opt
 ```
 
-##### Bash
+To build the `project-manager` one needs to launch `sbt` - one way to do it is
+to execute `./run backend sbt`. When in the _sbt prompt_ one can request
+compilation of the `project-manager`:
 
 ```bash
-sbt buildProjectManagerDistribution
-```
-
-##### PowerShell
-
-```powershell
-sbt.bat buildProjectManagerDistribution
+sbt:enso> buildProjectManagerDistribution
 ```
 
 When the command is completed, a development version of the project manager will
 have appeared in the `built-distribution` directory.
 
-The IDE will connect to the running project manager to look up the project and
-start the language server. The required version of the language server is
-specified in the `edition` field of the `package.yaml` project description. Enso
-projects are located in the `~/enso` directory on Unix and `%userprofile%\enso`
-on Windows systems by default.
+Project manager is there to wait for the IDE to connect to it and then launch
+the engine with its embedded language server. To build the engine issue
+following command in the _sbt prompt_:
 
 ```bash
-cat ~/enso/projects/Unnamed/package.yaml
+sbt:enso> buildEngineDistribution
 ```
 
-```yaml
```

Comment:
I do remember that setting all these editions, and symlinks was a pain when I was onboarding myself. I goal is to remove that from the `CONTRIBUTING.md` documentation altogether...